cognee/infrastructure/databases/vector/embeddings/DefaultEmbeddingEngine.py

```diff
@@ -0,0 +1,13 @@
+from typing import List
+from fastembed import TextEmbedding
+from .EmbeddingEngine import EmbeddingEngine
+
+class DefaultEmbeddingEngine(EmbeddingEngine):
+    async def embed_text(self, text: List[str]) -> List[float]:
```

Comment:
We should change the EmbeddingEngine protocol.

Reply:
We want to optimize embedding creation by sending multiple text strings at once. Sending string instead of the list is a leftover. `DefaultEmbeddingEngine` is the only class using `EmbeddingEngine` protocol.